purchase_variant_configurator_on_confirm/models/product_template.py

```diff
@@ -16,15 +17,14 @@ class ProductTemplate(models.Model):
     def _select_seller(self, partner_id=False, quantity=0.0,
                        date=time.strftime(DEFAULT_SERVER_DATE_FORMAT),
                        uom_id=False):
-        # this method is copied from the standard _select_seller on the
-        # product.product model
+        """ Copy from standard: `ProductProduct._select_seller` """
         self.ensure_one()
```

Comment:
Let's try a trick here, creating a "in-memory" product record and calling the method on it:
```
product = self.env['product.product'].new({
    'product_tmpl_id': self.id,
    'name': self.name,
})
return product._select_seller(
    partner_id=partner_id, quantity=quantity, date=date, uom_id=uom_id,
)
```